readme.rst

```diff
@@ -55,6 +55,17 @@ The command:
 the more standard ``make html``. For instance, ``PROJECT=azure make html`` will create the 
 azure related html pages in _build/azure.)
 
+Check compliance with contribution guides for each project 
+~~~~~~~~~~~
+Each project's documentation has its own contribution guide: 
```

Comment:
Let's say 'Each project has its own contribution guide:" since each project in this case is itself a documentation set, and we don't really have documentation of the documentation set.

Reply:
Ah, I meant the whole project (code + docs) has documentation that has a contribution guide. I see your point though, (the project is the documentation on its own). I have ammended.